components/proxy/src/main/java/com/hotels/styx/StyxServer.java

```diff
@@ -161,6 +164,8 @@ public StyxServer(StyxServerComponents config) {
     public StyxServer(StyxServerComponents components, Stopwatch stopwatch) {
         this.stopwatch = stopwatch;
 
+        initialiseHttpHeaderSanitiser(components.environment().configuration());
```

Comment:
I wouldn't initialise the HeaderSanitiser here as this method would probably not be invoked by users starting Styx programmatically.

Reply:
fixed, it's now in Environment